pkg/api/v1/pod/util.go

```diff
@@ -50,6 +50,47 @@ func FindPort(pod *v1.Pod, svcPort *v1.ServicePort) (int, error) {
 	return 0, fmt.Errorf("no suitable port for manifest: %s", pod.UID)
 }
 
+type ContainerType int
+
+const (
+	InitContainer = 1
+	Container = 2
+	EphemeralContainer = 4
+)
```

Comment:
I think we would probably also want an:
``
AllContainers = InitContainer | Container | EphemeralContainer
```